test_conformance/compiler/test_feature_macro.cpp

```diff
@@ -703,11 +766,25 @@ int test_consistency_c_features_list(cl_device_id deviceID,
     }
 
     log_info("\nSupported features based on queries to API/compiler :\n");
+
     for (auto each_f : vec_to_cmp)
     {
         log_info("%s\n", each_f.c_str());
     }
 
+    for (auto each_f : vec_to_cmp)
+    {
+        if (find(vec_device_feature_names.begin(),
+                 vec_device_feature_names.end(), each_f)
+            == vec_device_feature_names.end())
+        {
+            log_info("Comparison list of features - failed\n");
```

Comment:
Nice-to-have: Could we print the missing feature here?

Reply:
```suggestion
            log_info("Comparison list of features - failed - missing %s\n",
                     each_f.c_str());
```

Reply:
Done.